test/solidity/Facets/NonStandardSelectorsRegistryFacet.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Tests for the batch function of this contract are missing. To increase test coverage, it would make sense to add them.

Reply:
The batch test is on lines 35-48